posthog/api/person.py

```diff
@@ -23,6 +24,15 @@ def get_last_event(self, person: Person) -> Union[dict, None]:
         else:
             return None
 
+    def get_first_event(self, person: Person) -> Union[dict, None]:
+        if not self.context['request'].GET.get('include_last_event'):
```

Comment:
To avoid confusion I think we should rename both instances of `include_last_event` to `include_events` or something